src/services/io/podio/JEventSourcePODIOsimple.h

```diff
@@ -42,7 +42,7 @@ class JEventSourcePODIOsimple : public JEventSource {
     std::set<std::string> m_INPUT_EXCLUDE_COLLECTIONS;
     bool m_run_forever=false;
 
-    bool m_inflight = false; // is an event currently in flight (and therefore using the EventStore)?
+    std::thread::id processing_thread_id;  // defaults to a "no thread" value
 
 };
 
```

Comment:
I understand what you are doing and why. However, it is going to make people very nervous so we should get rid of it in the next JANA release if at all possible. 

My preferred solution to this problem is to simply set 
`event->SetSequential(true)` and/or `jana:limit_events_in_flight=1`. 

Of course, I'm pretty sure this will have the same problems with `nskip` as what we had here before, so this is another item for the next JANA release.